tools/src/wyvern/tools/tests/effectSeparation/client1.wyv

```diff
@@ -0,0 +1,2 @@
+import effectSeparation.file1
+unit
```

Comment:
To perform more thorough testing, instead of returning `unit` in the modules' "clients," call some method(s) from the imported module.